chia-wallet/src/primitives/cat.rs

```diff
@@ -0,0 +1,167 @@
+    01ff02ffff03ffff09ffff0cff29ff80ff3480ff5c80ffff01ff0101ff8080ff
+    0180ff8080ff018080ffff0109ffff01ff088080ff0180ffff010980ff018080
+    ff0180ffff04ffff02ffff03ffff09ff11ff5880ffff0159ff8080ff0180ffff
+    04ffff02ff26ffff04ff02ffff04ff0dffff04ff0bffff04ff17ff8080808080
+    80ff80808080808080ff0180ffff01ff04ff80ffff04ff80ff17808080ff0180
+    ffff02ffff03ff05ffff01ff04ff09ffff02ff56ffff04ff02ffff04ff0dffff
+    04ff0bff808080808080ffff010b80ff0180ff0bff7cffff0bff34ff2880ffff
+    0bff7cffff0bff7cffff0bff34ff2c80ff0580ffff0bff7cffff02ff32ffff04
+    ff02ffff04ff07ffff04ffff0bff34ff3480ff8080808080ffff0bff34ff8080
+    808080ffff02ffff03ffff07ff0580ffff01ff0bffff0102ffff02ff2effff04
+    ff02ffff04ff09ff80808080ffff02ff2effff04ff02ffff04ff0dff80808080
+    80ffff01ff0bffff0101ff058080ff0180ffff04ffff04ff30ffff04ff5fff80
+    8080ffff02ff7effff04ff02ffff04ffff04ffff04ff2fff0580ffff04ff5fff
+    82017f8080ffff04ffff02ff26ffff04ff02ffff04ff0bffff04ff05ffff01ff
+    808080808080ffff04ff17ffff04ff81bfffff04ff82017fffff04ffff02ff2a
+    ffff04ff02ffff04ff8204ffffff04ffff02ff76ffff04ff02ffff04ff09ffff
+    04ff820affffff04ffff0bff34ff2d80ffff04ff15ff80808080808080ffff04
+    ff8216ffff808080808080ffff04ff8205ffffff04ff820bffff808080808080
+    808080808080ff02ff5affff04ff02ffff04ff5fffff04ff3bffff04ffff02ff
+    ff03ff17ffff01ff09ff2dffff02ff2affff04ff02ffff04ff27ffff04ffff02
+    ff76ffff04ff02ffff04ff29ffff04ff57ffff04ffff0bff34ff81b980ffff04
+    ff59ff80808080808080ffff04ff81b7ff80808080808080ff8080ff0180ffff
+    04ff17ffff04ff05ffff04ff8202ffffff04ffff04ffff04ff78ffff04ffff0e
+    ff5cffff02ff2effff04ff02ffff04ffff04ff2fffff04ff82017fff808080ff
+    8080808080ff808080ffff04ffff04ff20ffff04ffff0bff81bfff5cffff02ff
+    2effff04ff02ffff04ffff04ff15ffff04ffff10ff82017fffff11ff8202dfff
+    2b80ff8202ff80ff808080ff8080808080ff808080ff138080ff808080808080
+    80808080ff018080
+    "
+);
+
+/// This is the puzzle hash of the [CAT2 standard](https://chialisp.com/cats) puzzle.
+pub static CAT_PUZZLE_HASH: [u8; 32] = hex!(
+    "
+    37bef360ee858133b69d595a906dc45d01af50379dad515eb9518abb7c1d2a7a
+    "
+);
+
+/// This is the puzzle reveal of the [CAT2 multi-issuance TAIL](https://chialisp.com/cats#multi) puzzle.
+pub static EVERYTHING_WITH_SIGNATURE_TAIL_PUZZLE: [u8; 41] = hex!(
+    "
+    ff02ffff01ff04ffff04ff02ffff04ff05ffff04ff5fff80808080ff8080ffff
+    04ffff0132ff018080
+    "
+);
+
+/// This is the puzzle reveal of the old [CAT1 standard](https://chialisp.com/cats) puzzle.
+///
+/// **Warning:**
+/// It is recommended not to use CAT1 for anything other than backwards compatibility (e.g. offer compression),
+/// due to security issues uncovered in an audit. You can read more about the vulnerability that prompted the creation
+/// of CAT2 in the [CATbleed Post Mortem](https://github.com/Chia-Network/post-mortem/blob/main/2022-08/2022-08-19-CATbleed.md).
+pub static CAT_PUZZLE_V1: [u8; 1420] = hex!(
```

Comment:
do we still need this?
It seems risky to just keep it around.

Reply:
For offer compression